### PR TITLE
✨  Add Bento versions (1.0) to validator specs for `amp-twitter` and `amp-facebook-like`

### PR DESCRIPTION
**Pull request by westonruter**
I believe these were missed in #34715 and #33335.

~This also adds the `npm:true` flags for `amp-facebook-like`, `amp-social-share`, and `amp-video`.~ See #34838 instead.

### CODE REVIEW

File: extensions/amp-twitter/validator-amp-twitter.protoascii [hunk 1]
```diff
@@ -20,6 +20,7 @@ tags: {  # amp-twitter
   extension_spec: {
     name: "amp-twitter"
     version: "0.1"
+    version: "1.0"
     version: "latest"
     requires_usage: EXEMPTED
     deprecated_allow_duplicates: true
```
Comment (caroqliu) [inline extensions/amp-twitter/validator-amp-twitter.protoascii:26]: When adding validations for 1.0 where 0.1 had these looser specifications (usage not required, duplicates allowed), please use `satisfies` and `excludes`. #33478 is an exemplary PR to reference which does exactly this.

**Review by caroqliu — COMMENTED** [review]
Than you for doing this! Just one comment. Please also copy the `validator-amp-*.{html,out}` files in `0.1/tests` over to `1.0/tests` with the relevant script version bumps.

**Review by caroqliu — COMMENTED** [review]
`amp-facebook-like` and `amp-twitter` LGTM. Please remove `amp-video` and `amp-social-share` for now, as they require additional changes to their directory structures before they are ready to have `npm` enabled.

**Review by caroqliu — APPROVED** [review]

**Comment by kristoferbaxter** [comment]
This is failing to build in current state.

```
[16:35:46] ERROR: Error: Build failed with 1 error:
error: Could not resolve "extensions/amp-facebook-like/1.0/component.js" 
```

File: build-system/compile/bundles.config.extensions.json [hunk 2]
```diff
@@ -363,7 +363,8 @@
     "version": "1.0",
     "latestVersion": "0.1",
     "options": {
-      "hasCss": true
+      "hasCss": true,
+      "npm": true
```
Comment (caroqliu) [inline build-system/compile/bundles.config.extensions.json:367]: This `npm: true` should be moved to `amp-facebook`. The `amp-facebook-*` components now rely on a single `Facebook` Preact component which lives inside the `amp-facebook` directory.

**Review by caroqliu — CHANGES_REQUESTED** [review]

File: build-system/compile/bundles.config.extensions.json (hunk 2 above)
Comment (westonruter) [reply build-system/compile/bundles.config.extensions.json]: Moved in 0a0f28e561e292b115792728a3e03196226839a6.

**Review by caroqliu — APPROVED** [review]

**Review by twifkak — APPROVED** [review]
Validator changes LGTM (from the wg-caching perspective). Sorry for the delay!

**Comment by westonruter** [comment]
CircleCI seems stuck.

**Pull request merged**